doc/class_property_descriptor.md

```diff
@@ -1,11 +1,93 @@
-# Class propertry and descriptor
+# Class property and descriptor
 
 Property descriptor for use with `Napi::ObjectWrap::DefineClass()`.
 This is different from the standalone `Napi::PropertyDescriptor` because it is
 specific to each `Napi::ObjectWrap<T>` subclass.
 This prevents using descriptors from a different class when defining a new class
 (preventing the callbacks from having incorrect `this` pointers).
 
+## Example
+
+```cpp
+#include <napi.h>
+
+class Example : public Napi::ObjectWrap<Example> {
+  public:
+    static Napi::Object Init(Napi::Env env, Napi::Object exports);
+    Example(const Napi::CallbackInfo &info);
+
+  private:
+    static Napi::FunctionReference constructor;
+    double _value;
+    Napi::Value GetValue(const Napi::CallbackInfo &info);
+    Napi::Value SetValue(const Napi::CallbackInfo &info);
+};
+
+Napi::Object Example::Init(Napi::Env env, Napi::Object exports) {
+    Napi::Function func = DefineClass(env, "Example", {
+        // Register a class instance accessor with getter and setter functions.
+        InstanceAccessor("value", &Example::GetValue, &Example::SetValue),
+        // We can also register a readonly accessor by passing nullptr as the setter.
+        InstanceAccessor("readOnlyProp", &Example::GetValue, nullptr)
+    });
+
+    constructor = Napi::Persistent(func);
+    constructor.SuppressDestruct();
+    exports.Set("Example", func);
+
+    return exports;
+}
+
+Example::Example(const Napi::CallbackInfo &info) : Napi::ObjectWrap<Example>(info) {
+    Napi::Env env = info.Env();
+    // ...
+    Napi::Number value = info[0].As<Napi::Number>();
+    this->_value = value.DoubleValue();
+}
+
+Napi::FunctionReference Example::constructor;
+
+Napi::Value Example::GetValue(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    return Napi::Number::New(env, this->_value);
+}
+
+Napi::Value Example::SetValue(const Napi::CallbackInfo &info, const Napi::Value &value) {
+    Napi::Env env = info.Env();
+    // ...
+    Napi::Number arg = value.As<Napi::Number>();
+    this->_value = arg.DoubleValue();
+    return this->GetValue(info);
+}
+
+// Initialize native add-on
+Napi::Object Init (Napi::Env env, Napi::Object exports) {
+    Example::Init(env, exports);
+    return exports;
+}
+
+// Regisgter and initialize native add-on
```

Comment:
Tupo here  **Regisgter**  should be **Register**

Reply:
Fixed here and where I copied it from :-)